xcode/Subconscious/Shared/Components/Settings/SettingsView.swift

```diff
@@ -51,21 +51,31 @@ struct GatewayProvisioningSection: View {
     var body: some View {
         Section(
             content: {
-                ValidatedFormField(
-                    placeholder: "Enter your invite code",
-                    field: app.state.inviteCodeFormField,
-                    send: Address.forward(
-                        send: app.send,
-                        tag: AppAction.inviteCodeFormField
-                    ),
-                    caption: "Look for this in your welcome email."
-                )
-                .autocapitalization(.none)
-                .autocorrectionDisabled(true)
-                .onDisappear {
-                    app.send(.setInviteCode(app.state.inviteCodeFormField.value))
+                if let gatewayId = app.state.gatewayId {
+                    LabeledContent(
+                        "Gateway ID",
+                        value: gatewayId
+                    )
+                    .lineLimit(1)
+                    .textSelection(.enabled)
```

Comment:
If we have already exchanged an invite code for a gateway ID, hide the invite code. This means that once you've redeemed one invite code you'll have to delete the app before you can redeem another, but it affords a cleaner UI for the more common case of having just one code.